src/data/meetings.js

```diff
@@ -53,7 +53,7 @@ export const WorkingGroups = [
   {
     name: 'Pangeo Forge',
     description: 'Coordinating development on Pangeo Forge.',
-    schedule: 'Every other Monday at 2pm ET',
+    schedule: 'Every other Wednesday at 2pm ET',
```

Comment:
the time is fine with me, but we might want to choose the offset week in a way that doesn't conflict with the [zarr community meetings](https://zarr.dev/community-calls/)

Reply:
We plan to start on Oct 23, so that should be good?